relay-server/src/services/processor.rs

```diff
@@ -2344,6 +2380,35 @@ impl EnvelopeProcessorService {
             }
         });
     }
+
+    fn app_features(&self, message: &EnvelopeProcessor) -> AppFeatures {
+        match message {
+            EnvelopeProcessor::ProcessEnvelope(v) => AppFeature::from(v.envelope.group()).into(),
+            EnvelopeProcessor::ProcessMetrics(_) => AppFeature::Unattributed.into(),
+            EnvelopeProcessor::ProcessBatchedMetrics(_) => AppFeature::Unattributed.into(),
```

Comment:
Should these two be `UnattributedMetrics`?

Reply:
No, this is actually a bit ugly.  They are unattributed because they are later updated with the "correct" weights, we only get these weights after we processed (parsed) the contained payload.

I kept the default here unattributed that if this shows up in the COGS somewhere we know that this shouldn't be there (either a bug or a small edge case), I tried explaining this in the `AppFeature::Unattributed` docs but maybe this is not clear enough.

Reply:
The explanation is good, I just was wondering about the default value.

Reply:
@Dav1dde could you add a comment here to point out that their weights will be updated later?